tests/conftest.py

```diff
@@ -180,6 +180,49 @@ def failing_prepare_commit_msg_repo(tempdir_factory):
         yield path
 
 
+@pytest.fixture
+def failing_post_checkout_repo(tempdir_factory):
+    path = git_dir(tempdir_factory)
+    config = {
+        'repo': 'local',
+        'hooks': [{
+            'id': 'post-checkout',
+            'name': 'Post checkout',
+            'entry': 'bash -c "exit 1"',
+            'language': 'system',
+            'always_run': True,
+            'stages': ['post-checkout'],
+        }],
+    }
+    write_config(path, config)
+    with cwd(path):
+        cmd_output('git', 'add', '.')
+        git_commit()
+        yield path
+
+
+@pytest.fixture
+def post_checkout_repo(tempdir_factory):
+    path = git_dir(tempdir_factory)
+    config = {
+        'repo': 'local',
+        'hooks': [{
+            'id': 'post-checkout',
+            'name': 'Post checkout',
+            'entry': 'bash -c "echo ${PRE_COMMIT_PREV_HEAD_REF}"',
+            'language': 'system',
+            'always_run': True,
+            'verbose': True,
+            'stages': ['post-checkout'],
+        }],
+    }
+    write_config(path, config)
+    with cwd(path):
+        cmd_output('git', 'add', '.')
+        git_commit()
+        yield path
```

Comment:
since these are only used once, it probably makes sense to inline them adjacent to the test itself

Reply:
Do you want me to inline the entire fixture or just the `git add` / `git commit` part of this fixture?

Reply:
ah mostly that there isn't a reason to put it in `conftest.py` unless it is being shared across multiple tests

Reply:
Sounds good to me, I will move it.